packages/legacy/package.json

```diff
@@ -59,33 +60,33 @@
     "@storybook/theming": "^7.3.2",
     "@sveltejs/vite-plugin-svelte": "^2.4.2",
     "@types/prismjs": "^1.26.0",
-    "@typescript-eslint/eslint-plugin": "^6.4.0",
-    "@typescript-eslint/parser": "^6.4.0",
-    "@viamrobotics/eslint-config": "^0.1.2",
-    "@viamrobotics/prettier-config": "^0.1.2",
-    "@viamrobotics/prime-core": "workspace:../core",
-    "@viamrobotics/typescript-config": "^0.0.4",
+    "@typescript-eslint/eslint-plugin": "^6.7.0",
+    "@typescript-eslint/parser": "^6.7.0",
+    "@viamrobotics/eslint-config": "^0.2.4",
+    "@viamrobotics/prettier-config": "^0.3.2",
+    "@viamrobotics/prime-core": "workspace:*",
```

Comment:
Is this correct? Why not link to the core package directly?

Reply:
It is! with `workspace:*`, pnpm understands how to sort dependencies when running `pnpm run -r`. With `workspace:../core`, it doesn't, for some reason.

This is important, because `core`'s build _must_ run before `blocks` et. al. or things won't work